app/src/main/java/id/rizmaulana/covid19/ui/overview/DashboardActivity.kt

```diff
@@ -109,6 +117,21 @@ class DashboardActivity : BaseActivity() {
 
     }
 
+    private fun handlePinnedUpdate(data: CovidDetail?) {
+        with(binding.countryInfo) {
+            data?.let {detail ->
+                val header = StringBuilder().apply {
+                    append(detail.locationName)
+                    append(", Updated: ${NumberUtils.formatTime(detail.lastUpdate)}")
```

Comment:
why you decide to use `StringBuilder()` ? for simplifying purpose, you can use `.plus(...)` instead of creating a new object for simple thing.
```kt
val lastUpdate = NumberUtils.formatTime(detail.lastUpdate)
val header = detail.locationName
             .plus(getString(R.string.updated, lastUpdate))
```

Reply:
I think both implementation are the same, and adding `.plus` is actually a little bit harder to read as `.plus` is actually an operator, thus if you speak about simplicity, using `+` is much simpler. Changing it to `+` with extracted string

Reply:
and yes, maybe you should the primitive way with `+` instead of make a new object.